tests/integration/targets/sysrc/tasks/main.yml

```diff
@@ -164,6 +167,10 @@
           - not sysrc_example4_idempotent.changed
           - "'nginx_enable=\"YES\"' in sysrc_example4_content.stdout_lines"
 
+    - name: Stop and remove testjail
+      when: is_testjail_up.rc == 1
+      command: "ezjail-admin delete -wf testjail"
```

Comment:
It's probably best to use a `block:` (starting with the `include` you added above) and put this task this in the `always:` part, so it is also executed in case something above fails.

Reply:
Updated to use a block with always